index.html

```diff
@@ -223,6 +223,7 @@ <h2 class="intro">
             placeholder="Email address"
             required
           />
+          <small id="email-warning"></small>
```

Comment:
- [ ] You did a really great work the only missing part is as the requirements say the error messages should be at the bottom near the submit button.
` you show an error message to the user near the submit button informing them of the error and the form is not sent.`

![image](https://user-images.githubusercontent.com/35535130/156428738-823f027e-6919-4ba6-8273-43aaa0842ac9.png)

Reply:
Thanks for the feedback 😎